src/shop-cart.html

```diff
@@ -14,7 +14,7 @@
 <link rel="import" href="shop-form-styles.html">
 
 <dom-module id="shop-cart">
-
+  <link rel="lazy-import" href="lazy-resources.html" group="imported-by-app">
```

Comment:
Add a comment, and maybe a todo to restructure lazy imports to be loaded more locally.